src/include/execution/vm/bytecode_handlers.h

```diff
@@ -34,6 +34,11 @@
 #define VM_OP_WARM VM_OP inline
 #define VM_OP_COLD VM_OP NEVER_INLINE
 
+namespace terrier::execution::vm {
+STRONG_TYPEDEF(col_id_t, uint16_t);
+STRONG_TYPEDEF(col_id_32_t, uint32_t);
```

Comment:
Shouldn't need a separate type for this. We can cast as needed.

Reply:
I have fix this.